Makefile

```diff
@@ -4,6 +4,9 @@ export GO111MODULE        := on
 export PATH               := .bin:${PATH}
 
 format: .bin/ory node_modules  # formats all source code
+	make format-fast
+
+format-fast:
```

Comment:
I assume this adds a faster formatting run target? 
If so, what do you think about making it the default (so having it triggered by the `make format` target) and then making the current format job live under `make format-full`?

Reply:
True, we could also just ignore npm i if the formatter tasks works. But let's track this in a separate issue :)

Reply:
https://github.com/ory/docs/issues/1110